### PR TITLE
update firefox-profile to 1.0.0

### PR DESCRIPTION
**Pull request by andriilazebnyi**

### CODE REVIEW

File: lib/firefox-profile-launch-service.js [hunk 1]
```diff
@@ -49,7 +49,12 @@ class FirefoxProfileLaunchService {
 
     buildExtension () {
         return new Promise((resolve) => {
-            this.profile.encoded((zippedProfile) => {
+            this.profile.encoded((err, zippedProfile) => {
+                if (err) {
+                    console.error(`Failed to encode Firefox profile: ${err}`)
+                    return resolve()
```
Comment (christian-bromann) [inline lib/firefox-profile-launch-service.js]: it should reject the promise in case of an error

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "url": "https://github.com/webdriverio/wdio-firefox-profile-service/issues"
   },
   "dependencies": {
-    "firefox-profile": "0.4.9"
+    "firefox-profile": "^1.0.0"
```
Comment (christian-bromann) [inline package.json]: let's specify version with `~1.0.0` just to be safe

**Review by christian-bromann — CHANGES_REQUESTED** [review]
some comments

**Comment by andriilazebnyi** [comment]
@christian-bromann Thanks. Fixed.

File: lib/firefox-profile-launch-service.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ class FirefoxProfileLaunchService {
             this.profile.encoded((err, zippedProfile) => {
                 if (err) {
                     console.error(`Failed to encode Firefox profile: ${err}`)
-                    return resolve()
+                    return reject()
```
Comment (christian-bromann) [inline lib/firefox-profile-launch-service.js]: reject is not defined, please define it

**Review by christian-bromann — CHANGES_REQUESTED** [review]

**Comment by christian-bromann** [comment]
@andriilazebnyi there is still a linting error

**Comment by andriilazebnyi** [comment]
@christian-bromann Looks like VSCode eslint plugin didn't catch it. Fixed.

**Review by christian-bromann — APPROVED** [review]
👍🏻

**Pull request merged**